src/i18n/en.js

```diff
@@ -272,6 +272,20 @@ export default {
     header_title: `Subscriber General Activity Report`,
     page_description: `Through this report you will be able to know the general activity of a specific subscriber`,
     page_title: `Doppler | Subscriber History`,
+    search_form: {
+      aria_label: `Filters form to search subscriber history`,
```

Comment:
We try to use caps on Doppler custom fields or special features like `Subscriber History`, `Email` `Last Name`, `First Name` in english and spanish.